frontend/src/components/ContextMenu.css

```diff
@@ -16,8 +16,16 @@
   z-index: 1;
 }
 
+.smart-hub--context-menu__left_and_up {
```

Comment:
Can't combine the `left` and `up` CSS classes because the `transform` attribute overwrites itself, hence the `left_and_up` class